### PR TITLE
rb-aes*: updates for Ruby ports

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

1. Rename current `rb-aes` to `rb-aescrypt`, since it is unrelated to `rb-aes` gem. Fix the build which is presently broken.
2. Add `rb-aes` and `rb-aes_key_wrap` ports.
3. Update `rb-crypt-isaac` (it is no longer noarch).

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.3.1
Xcode 15.3

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: ruby/rb-aescrypt/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           ruby 1.0
+
+ruby.branches       3.3 3.2 3.1
+ruby.setup          {aescrypt aes-rb} 0.1.0 extconf.rb {README \
+                    aes-example.rb aes-test.rb aes-ecb-tbl.txt}
+revision            1
```
Comment (herbygillot) [inline ruby/rb-aescrypt/Portfile:9]: Is this a new port? If so, `revision` should be starting at zero

File: ruby/rb-aescrypt/Portfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+maintainers         nomaintainer
+description         An AES encryption algorithm for Ruby, using the AESCrypt library
+long_description	An all-C (for performance) implementation of the AES \
```
Comment (herbygillot) [inline ruby/rb-aescrypt/Portfile:12]: There may be an actual tab here that needs to be expanded, as per the modeline.

File: ruby/rb-crypt-isaac/Portfile [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
 
-ruby.setup		crypt-isaac 0.9.1 gem {} rubygems
-maintainers		nomaintainer
-description		Implementation of the ISAAC psuedo random number generator.
-long_description	This is a Ruby implementation of the ISAAC \
+PortSystem          1.0
+PortGroup           ruby 1.0
+
+ruby.branches       3.3 3.2 3.1
+ruby.setup          crypt-isaac 1.2.0 gem {} rubygems
```
Comment (herbygillot) [inline ruby/rb-crypt-isaac/Portfile:7]: `license` needs to be set here, as per the output of `port lint`.

File: ruby/rb-aescrypt/Portfile (hunk 2 above)
Comment (barracuda156) [reply ruby/rb-aescrypt/Portfile:12]: @herbygillot Thanks, I will fix it.

**Comment by barracuda156** [comment]
@herbygillot Fixed lint issues now.

**Pull request merged**